packages/commerce-sdk-react/src/hooks/ShopperContexts/query.ts

```diff
@@ -1,20 +1,46 @@
 /*
- * Copyright (c) 2022, Salesforce, Inc.
+ * Copyright (c) 2023, Salesforce, Inc.
  * All rights reserved.
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-import {NotImplementedError} from './../utils'
+import {UseQueryResult} from '@tanstack/react-query'
+import {ApiClients, ApiQueryOptions, Argument, DataType} from '../types'
+import useCommerceApi from '../useCommerceApi'
+import {useQuery} from '../useQuery'
+import {mergeOptions} from '../utils'
+import * as queryKeyHelpers from './queryKeyHelpers'
+
+type Client = ApiClients['shopperContexts']
 
 /**
- * WARNING: This method is not implemented.
- *
  * A hook for `ShopperContexts#getShopperContext`.
- * Gets the shopper's context based on the shopperJWT.
+ * Gets the shopper's context based on the shopperJWT. ******** This API is currently a work in progress, and not available to use yet. ********
```

Comment:
The js comments like these are usually not updated often and certainly not in the process when we release API changes. I wonder if there is a better place for such notes. Just a nit worry that we might forgot to remove it in the future.